core/src/main/java/org/apache/brooklyn/core/BrooklynFeatureEnablement.java

```diff
@@ -61,6 +61,36 @@
     /** whether feeds are automatically registered when set on entities, so that they are persisted */
     public static final String FEATURE_FEED_REGISTRATION_PROPERTY = FEATURE_PROPERTY_PREFIX+".feedRegistration";
 
+    /**
+     * <p>
+     * Enabling CORS is strongly discouraged.
```

Comment:
The point is that you can't tell at this level whether CORS is appropriate or not; that's more an architectural question. Sometimes CORS is a proper solution to a given need to integrate multiple server backends onto a page in a web application (e.g. with a plugin based web application).  Sometimes it is a hack to glue in some REST call that you should really be handling server side.  We can't simply say 'use of CORS is discouraged'.